frontend/src/lib/components/ActivationSidebar/ActivationSidebar.tsx

```diff
@@ -0,0 +1,126 @@
+        id,
+        fullWidth: true,
+        type: 'secondary',
+        icon: completed ? <IconCheckmark /> : skipped ? <IconClose /> : null,
+        status: completed ? 'primary-alt' : skipped ? 'muted' : undefined,
+        tooltip: name,
+    }
+    if (url) {
+        params.to = url
+        params.targetBlank = true
+    } else {
+        params.onClick = () => runTask(id)
+    }
+    return (
+        <li>
+            {displaySideAction ? (
+                <LemonButtonWithSideAction
+                    {...params}
+                    sideAction={{
+                        icon: <IconClose />,
+                        tooltip: 'Skip task',
+                        onClick: () => skipTask(id),
+                        status: 'muted',
+                    }}
+                >
+                    {content}
+                </LemonButtonWithSideAction>
+            ) : (
+                <LemonButton {...params}>{content}</LemonButton>
+            )}
+        </li>
+    )
+}
+
+const ActivationSidebar = (): JSX.Element => {
+    const { isActivationSideBarShown } = useValues(navigationLogic)
+    const { hideActivationSideBar } = useActions(navigationLogic)
+    const { activeTasks, completedTasks, completionPercent, showSessionRecordingConfig } = useValues(activationLogic)
+    const { setShowSessionRecordingConfig } = useActions(activationLogic)
+
+    return (
+        <div className={clsx('ActivationSideBar', !isActivationSideBarShown && 'ActivationSideBar--hidden')}>
+            <div className="ActivationSideBar__content px-4 pb-16">
+                <div className="ActivationSideBar__close_button">
+                    <LemonButton
+                        icon={<IconClose />}
+                        onClick={() => {
+                            if (showSessionRecordingConfig) {
+                                setShowSessionRecordingConfig(false)
+                            } else {
+                                hideActivationSideBar()
+                            }
+                        }}
+                    />
+                </div>
+                {showSessionRecordingConfig ? (
+                    <SessionRecordingConfig />
+                ) : (
+                    <>
+                        <h2 className="subtitle">Quick Start</h2>
+                        <p>Learn how to get the most out of PostHog, straigth away.</p>
+                        <div className="my-4 flex items-center justify-center">
+                            <div className="flex flex-col items-center">
+                                <Progress
+                                    type="circle"
+                                    strokeWidth={10}
+                                    percent={completionPercent}
+                                    format={() => activeTasks.length}
+                                />
+                                <p className="text-muted mt-2">Remaining tasks</p>
```

Comment:
It may be worth taking the word 'task' out, just so it doesn't feel so choreish?

"Still to go" maybe?